test/index.js

```diff
@@ -8,8 +8,23 @@ import {Application} from 'spectron';
 let app;
 
 test.before(async () => {
+  let pathToBinary;
```

Comment:
I don't feel strongly about it, but `binaryPath` seems more descriptive and active (vs passive).